src/index.html

```diff
@@ -20,8 +23,389 @@
       href="styles/index.scss"
     />
   </head>
+  <body class="page__body">
+    <header class="header">
+      <a
+        class="header__logo-link"
+        href="index.html"
+      >
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="MOYO logo"
+        />
+      </a>
 
-  <body>
-    <h1>Catalog</h1>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="iMac photo"
+          class="card__photo"
+        />
+        <div class="card__info">
+          <h2 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__grade">Reviews: 5</div>
```

Comment:
```suggestion
            <p class="card__grade">Reviews: 5</p>
```
in the previous review mentor suggested using `p` tag around a common text
It was related to all similar cases - that's a general rule (always fix all similar cases) - take into account in the future

Reply:
Done. Thank you.